### DISCUSSION TITLE
Passwordless login (code via email)

### DISCUSSION THREAD

**Discussion by seniorquico** [ory/kratos#1319] [Q&A] [answered]
We're looking to migrate off Auth0, but we use the passwordless login flow-- specifically, short-lived codes sent through email. I found a few references in the documentation to ORY Kratos supporting passwordless login. However, I cannot find any examples or hints in the code base. Is this supported at this time?

**Comment by Benehiko** [comment, marked as answer]
Hi @seniorquico 

Thank you for your interest in Ory :) 

Currently passwordless login is not supported. 
You can track the issue here https://github.com/ory/kratos/issues/26

**Reply by seniorquico** [reply]
Yes, I was beginning to wonder if this was the case. I've noticed several things mentioned in the docs that are quite there yet. No worries, gives me a good idea of what's planned to come! 😄

**Comment by ysaakpr** [comment]
I believe the requirement itself is different from the issue #26 which is more towards MFA. This is for passwordless login, not about MFA. passwordless login can exist with out MFA capabilities. For example there are cases where i have to allow user to create an account or login to account just using an `email and otp send to email` or `mobile number and otp send as sms` combo to verify the user. Are there any plan to add passwordless as a default method of identity verification apart from the MFA capabilities.

If #26  is handling it, let me know how one can achieve it, since #26 is now closed

**Reply by Benehiko** [reply]
Hi @ysaakpr
yes you are right, I linked it due to it being an umbrella issue that also mentioned passwordless. Thank you for providing more links to the proper issues below :pray:

**Comment by ysaakpr** [comment]
#1570 #2029 are referencing to similar kind of issues, tagged here for better discussions
